backend/.env

```diff
@@ -0,0 +1,6 @@
+NODE_ENV=development
```

Comment:
@SushantBabu97 i think you forget to add .env in gitignore

Reply:
yep. fixed it